app/posts/manage-teacher-training-applications/2020-11-09-changing-an-offer-iteration.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Changing an offer iteration
+description: Improvements to the way users edit an offer
+date: 2020-11-09
+---
+
+[View current flow](/manage-teacher-training-applications/changing-an-offer-november-2020).
```

Comment:
```suggestion
This is an interaction of [the current flow](/manage-teacher-training-applications/changing-an-offer-november-2020), which changes [x] because of [y].
```

Reply:
Add a little more context around how this iteration relates to the current flow.